frontend/javascripts/oxalis/view/settings/export_bounding_box_modal.js

```diff
@@ -31,12 +32,15 @@ const ExportBoundingBoxModal = ({ destroy, dataset, boundingBox }: Props) => {
   };
 
   const layerNames = dataset.dataSource.dataLayers.map(layer => {
-    const nameIfColor = layer.category === "color" ? layer.name : null;
+    const nameIfFromDataset = layer.category === "color" || !hasVolumeTracing ? layer.name : null;
     const nameIfVolume =
-      layer.category === "segmentation" && layer.fallbackLayerInfo && layer.fallbackLayerInfo.name
+      hasVolumeTracing &&
+      layer.category === "segmentation" &&
+      layer.fallbackLayerInfo &&
+      layer.fallbackLayerInfo.name
```

Comment:
```suggestion
      layer.fallbackLayerInfo != null
```

should be enough, I think, since `name` should always exist given that the object is not null.